winreg.ts

```diff
@@ -0,0 +1,39 @@
+///<reference path=".d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+let Registry = require("winreg");
```

Comment:
I'll try to prepare `winreg.d.ts` and add it here and send PR to DefinitelyTyped. However this will require some time, as I'll have to add unit tests and read they contribution guidelines. Can we live this way for the moment?